tests/ecmascript/test-bi-math-log2.js

```diff
@@ -0,0 +1,44 @@
+/*
+ *  Math.log2()
+ */
+
+/*@include util-number.js@*/
+
+/*===
+function true false true
+1
+NaN
+NaN
+-Infinity
+-Infinity
+0
+Infinity
+1
+10
+812
+9965784
+===*/
+
+function test() {
+    var pd = Object.getOwnPropertyDescriptor(Math, 'log2');
+    print(typeof pd.value, pd.writable, pd.enumerable, pd.configurable);
+    print(Math.log2.length);
+
+    printExact(Math.log2(NaN));
+    printExact(Math.log2(-1));
+    printExact(Math.log2(0));
+    printExact(Math.log2(-0));
+    printExact(Math.log2(1));
+    printExact(Math.log2(Infinity));
+
+    printExact(Math.log2(2));
+    printExact(Math.log2(1024));
+    printExact(Math.log2(2.73121871170759e+244));
+    printRounded6(Math.log2(1000));
```

Comment:
One interesting test for log2 would be exactness for inputs like 2^n where n is in a reasonable range (at least 0 to 53, i.e. the exact integer range).

I know no guarantees are in the specification but for practical purposes this is probably quite interesting. The guarantee holds for V8 at least for the entire IEEE double exponent range.